Signal/src/call/WebRTCCallMessageHandler.swift

```diff
@@ -33,34 +33,22 @@ public class WebRTCCallMessageHandler: NSObject, OWSCallMessageHandler {
 
     // MARK: - Call Handlers
 
-    public func receivedOffer(_ offer: OWSSignalServiceProtosCallMessageOffer, from callerId: String) {
+    public func receivedOffer(_ offer: SSKProtoCallMessageOffer, from callerId: String) {
         SwiftAssertIsOnMainThread(#function)
-        guard offer.hasId() else {
```

Comment:
These fields are required.

Reply:
> though see my comments in #3767 re: optional primitives. I'm not sure we're properly requiring them.

Yup, see my comments there.  We don't have a perfect solution - very happy to discuss alternatives.